mne/utils/testing.py

```diff
@@ -402,3 +402,14 @@ def dec(*args, **kwargs):
                 raise SkipTest(msg)
             raise
     return dec
+
+
+def assert_and_remove_boundary_annot(annotations, n=1):
```

Comment:
wouldn't you find it better to have 2 separated functions? 

```py
def assert_boundary_annotations(annotations, n_boundaries=1):
..
def _remove_boundary_annotations(annotations):
```

Reply:
Not so far, they are always used together in testing so it saves lines (DRY) to keep them together. We can separate later if necessary.